docs/content/backup_restore.md

```diff
@@ -60,31 +60,25 @@ Note that using this option, backup files will never be cleaned up.
 
 ### Extra options
 
-If you temporarily want to provide your own options to the backup binary,
-use the `--extra-options` (`-o`) in your command:
+If you need to occasionally provide some custom options to the backup binary,
+use the `--extra-options` (`-o`) option in your command:
 
 ```sh
 console db-tools:backup --extra-options "--opt1 val1 --opt2 val2 --flag"
 ```
 
-If you always need to use the same custom options when backing up, you can
-define them as default options in the bundle configuration file:
+By default, those custom options will be added to the [default options](./configuration#default-binary-options)
+defined in the bundle configuration file. Use `--replace-defaults` (`r`) if you
+need to override the defaults:
 
-```yaml
-# config/packages/db_tools.yaml
-db_tools:
-    # ...
-    backupper_options:
-        my_doctrine_connection: "--opt1 val1 --opt2 val2 --flag"
+```sh
+console db-tools:backup --replace-defaults -o "--opt1 val1 --opt2 val2 --flag"
 ```
 
-See the section about the [bundle configuration](./configuration#default-binary-options)
-for more information.
-
 :::warning
-The `--extra-options` option overrides default options defined in the
-configuration file as well as those from the bundle itself. (No merge
-will be performed.)
+Whether the `--replace-defaults` option is used or not and whether default
+options are defined in the configuration file or not, `--extra-options` option
+value will always override default options from the bundle.
```

Comment:
I don't understand this sentence? are you sure about it?

Reply:
Hum ok I think I see what you mean. I understand why this operate like this (because I know the code behind it), but It's not easy to understand.. :/

Reply:
Maybe Backupper and Restorer could have 2 properties: $defaultOptions and $extraOptions ?